src/server/controllers/sendChangePasswordEmail.ts

```diff
@@ -252,6 +252,23 @@ const sendEmailInOkta = async (
       }),
     );
   } catch (error) {
+    // handle check for if the user is not found
+    // this is a special case because we don't want to send an email
+    // but we want to show the email sent page
+    // so the reset password page cannot be used for account enumeration
+    if (
+      isOktaError(error) &&
+      error.status === 404 &&
+      error.code === 'E0000007'
```

Comment:
do we have these codes mapped to an enum (or similar somewhere?) would be nice to have them a little bit more human readable.. e.g: `OktaErrorCodes.ResourceNotFoundError` for this one

Reply:
There is

https://github.com/guardian/gateway/blob/a3801031c60d067675d92de30e82a869a7d85a7a/src/server/models/okta/Error.ts#L29-L37

But it's not quite useful as a typescript enum, so maybe thats worth looking into for a future improvement PR!

Reply:
will make a ticket for the 'ol backlog